tests/test_steps.py

```diff
@@ -83,33 +83,18 @@ def test_dependency_filtering():
         "c": {"b", "d"},
         "b": {"a"},
     }
-    assert filter_to_subgraph(dag, ["b"]) == {
+    assert filter_to_subgraph(dag, ["b"], downstream=True) == {
         "d": set(),
         "c": {"b", "d"},
         "b": {"a"},
         "a": set(),
     }
-    assert filter_to_subgraph(dag, ["b"], incl_forward=False) == {
+    assert filter_to_subgraph(dag, ["b"], downstream=False) == {
         "b": {"a"},
         "a": set(),
     }
 
 
-@patch("etl.steps.parse_step")
```

Comment:
I didn't see this before, but it covers the same case as one of the tests I added in the last PR, so not needed